src/test/functional/features/admin/admin-login.feature

```diff
@@ -5,19 +5,28 @@ Feature: Login/Logout
     And I am on FACT homepage
     Then I am logged out if I am a super admin
 
-  Scenario Outline: Login Unsuccessful
+#  Scenario Outline: Login Unsuccessful
```

Comment:
do we want to remove this?

Reply:
we need this scenario, it was committed mistakenly. Thanks